<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ventanaInicio</class>
 <widget class="QMainWindow" name="ventanaInicio">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>NIVELES</string>
  </property>
  <property name="styleSheet">
   <string notr="true">.btnFacil {
   background: #d30000;
   border-radius: 20px;
   border: nonepx;
   color: white;
}
.btnFacil:hoover {
   background-color: rgb(150, 0, 0);
}
.btnFacil:pressed{
   background-color: rgb(100, 0, 0)
}
.btnMedio {
   background: #d30000;
   border-radius: 20px;
   border: nonepx;
   color: white;
}
.btnMedio:hoover {
   background-color: rgb(150, 0, 0);
}
.btnMedio:pressed{
   background-color: rgb(100, 0, 0)
}
.btnDificil {
   background: #d30000;
   border-radius: 20px;
   border: nonepx;
   color: white;
}
.btnDificil:hoover {
   background-color: rgb(150, 0, 0);
}
.btnDificil:pressed{
   background-color: rgb(100, 0, 0)
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1000</width>
      <height>700</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1000</width>
      <height>700</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1000</width>
      <height>700</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_titulo_nivel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>1000</width>
       <height>80</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>1000</width>
       <height>80</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1000</width>
       <height>80</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 20pt &quot;Cooper Black&quot;;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#d30000;&quot;&gt;NIVELES&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnFacil">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>230</y>
       <width>140</width>
       <height>60</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;Cooper Black&quot;;</string>
     </property>
     <property name="text">
      <string>FACIL</string>
     </property>
     <property name="class" stdset="0">
      <string>btnFacil {
   background: #d30000;
   border-radius: 20px;
   border: nonepx;
   color: white;
}
btnFacil:hoover {
   background-color: rgb(150, 0, 0);
}
btnFacil:pressed{
   background-color: rgb(100, 0, 0)
}</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnMedio">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>230</y>
       <width>140</width>
       <height>60</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;Cooper Black&quot;;</string>
     </property>
     <property name="text">
      <string>MEDIO</string>
     </property>
     <property name="class" stdset="0">
      <string>btnMedio {
   background: #d30000;
   border-radius: 20px;
   border: nonepx;
   color: white;
}
btnMedio:hoover {
   background-color: rgb(150, 0, 0);
}
btnMedio:pressed{
   background-color: rgb(100, 0, 0)
}</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnDificil">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>230</y>
       <width>140</width>
       <height>60</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;Cooper Black&quot;;</string>
     </property>
     <property name="text">
      <string>DIFICIL</string>
     </property>
     <property name="class" stdset="0">
      <string>btnDificil {
   background: #d30000;
   border-radius: 20px;
   border: nonepx;
   color: white;
}
btnDificil:hoover {
   background-color: rgb(150, 0, 0);
}
btnDificil:pressed{
   background-color: rgb(100, 0, 0)
}</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
